### PR TITLE
Update documentation 2020-08-07

### PR DESCRIPTION
**Pull request by itays123**
- updated text example
- added babel example
- update installation command
- requirements section moved

### CODE REVIEW

**Comment by itays123** [comment]
All conflicts resolved!

File: README.md [hunk 1]
```diff
@@ -86,15 +91,46 @@ div { background: red; }
...
+### Babel
+Bundler transforms file content with presets and plugins with babel to ```javascript``` using Babel. <br />
+Before
+```ts
+/* index.ts */
+const fn = () => 1;
 ```
 After
 ```js
-/* 8277fbd0-903e-4a4b-87a7-cfa876924c7a.js */
```
Comment (itays123) [inline README.md:92]: this code is in line 95 now

File: README.md [hunk 2]
```diff
@@ -86,15 +91,46 @@ div { background: red; }
...
+### Text
+Bundler transforms file content to string export module.
+```txt
+/* hello.txt */
+Hello World!
 ```
+After
 ```js
-import styles from "./8277fbd0-903e-4a4b-87a7-cfa876924c7a.js"
-console.log(styles) // div { background: red; }
```
Comment (itays123) [inline README.md:97]: this code needs to be re-written, it got deleted

File: README.md [hunk 3]
```diff
@@ -15,6 +15,10 @@ import { foo } from "bar" // Deno Error
 
 Bundler bundles deno syntax files (url imports, .ts extensions) for the web.
 
+### No Requirements
+- no need for a cache folder in your project. Bundler uses the Deno cache system.
+- no need for a `node_modules` folder and a `package.json` file.
```
Comment (timreichen) [inline README.md:20]: No need for mentioning node and package.json. This is standard for deno.
```cache``` might be better mentioned under features.

File: README.md [hunk 4]
```diff
@@ -86,15 +91,46 @@ div { background: red; }
...
 ```
+After
 ```js
-import styles from "./8277fbd0-903e-4a4b-87a7-cfa876924c7a.js"
-console.log(styles) // div { background: red; }
+/* after bundling */
+export default 'Hello World!'
+```
+
+## Installation
```
Comment (timreichen) [inline README.md:125]: Installation and Usage is under Bundler CLI

File: README.md [hunk 5]
```diff
@@ -86,15 +91,46 @@ div { background: red; }
 ```
 ```js
 import styles from "./styles.css"
+
+console.log(styles) // div { background: red; }
+```
+
+### Babel
```
Comment (timreichen) [inline README.md:98]: Babel is not part of Bundler CLI but as a plugin that can be used with Bundler API. So maybe a plugin section for babel and text plugins.

**Pull request closed without merging**